### PR TITLE
Bug 1246689 - Add disclosure indicator to account profile when logged in

### PR DESCRIPTION
**Pull request by vbudhram**
This fixes https://github.com/mozilla/fxa-bugzilla-mirror/issues/333 by adding a disclosure indicator when a user is logged in.

### CODE REVIEW

File: Client/Frontend/Settings/AppSettingsOptions.swift [hunk 1]
```diff
@@ -284,16 +287,16 @@ class AccountStatusSetting: WithAccountSetting {
 
         if let account = profile.getAccount() {
             switch account.actionNeeded {
-            case .needsVerification:
+            case .none, .needsVerification:
                 var cs = URLComponents(url: account.configuration.settingsURL, resolvingAgainstBaseURL: false)
                 cs?.queryItems?.append(URLQueryItem(name: "email", value: account.email))
                 viewController.url = try! cs?.asURL()
```
Comment (st3fan) [inline Client/Frontend/Settings/AppSettingsOptions.swift]: This `try!` cannot be here. It will crash the app if an exception happens. Same below.

**Comment by farhanpatel** [comment]
### SwiftLint found issues

#### Warnings

File | Line | Reason |
| --- | ----- | ----- |
AppSettingsOptions.swift | 294 | if,for,while,do statements shouldn't wrap their conditionals in parentheses. 
AppSettingsOptions.swift | 306 | if,for,while,do statements shouldn't wrap their conditionals in parentheses. 
AppSettingsOptions.swift | 74 | Colons should be next to the identifier when specifying a type and next to the key in dictionary literals. 
AppSettingsOptions.swift | 594 | Colons should be next to the identifier when specifying a type and next to the key in dictionary literals. 
AppSettingsOptions.swift | 187 | Arguments can be omitted when matching enums with associated types if they are not used. 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by vbudhram** [comment]
@st3fan Updated per your feedback, r?

**Comment by vbudhram** [comment]
from st3fan: merge and tag for 8.3.

**Pull request merged**